src/main/java/sirius/web/services/SharedParametersAdapter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.services;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import sirius.kernel.health.Exceptions;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Simplifies implementing {@link SharedParameters} by providing facilities to copy {@link Parameter} descriptions from
+ * the {@link #dummyWithParameterAnnotations()} method.
+ */
+public abstract class SharedParametersAdapter implements SharedParameters {
+
+    private final List<Parameter> parameters = new ArrayList<>();
+
+    protected SharedParametersAdapter() {
+        try {
+            parameters.addAll(Arrays.stream(getClass().getMethod("dummyWithParameterAnnotations")
+                                                      .getAnnotationsByType(Parameter.class)).toList());
+        } catch (Exception e) {
+            Exceptions.handle(e);
+        }
+    }
+
+    /**
+     * A dummy method that is only used for annotating {@link Parameter} descriptions. Overwrite it with an empty body
+     * and add the shared {@link Parameter} annotations.
+     */
+    public abstract void dummyWithParameterAnnotations();
```

Comment:
Can't we have a better name for it? Something like inheritDescriptions or the like?

Reply:
`inheritDescriptions` implies that the method does anything at all. It doesn't, though, as it is a dummy. 🙃

Reply:
But then why abstract forcing classes to always overwrite it?

Reply:
The centralized abstract code provides means to collect parameter annotations from this dummy method. But there are none.

Annotations are attached by inheriting classes overwriting the method _and_ attaching the annotations. It does not make sense to extend the adapter else.

Of course, one could generate the parameter list differently as well, by implementing the interface and ignoring the adapter. (Via anonymous parameter classes for instance, which would soon create the need for a parameter adapter.) But grabbing the annotations from a dummy is likely the most simple solution.